hw/ip_templates/rv_plic/fpv/tb/coverage_waivers.tcl.tpl

```diff
@@ -18,3 +18,12 @@ check_cov -waiver -add -start_line 25 -end_line 56 -type {statement} -instance\
 # The port data_o in u_data_chk is untied and used nowhere.
 check_cov -waiver -add -start_line 25 -end_line 81 -type {statement} -instance\
  {dut.u_reg.u_chk.u_chk} -comment {data_o is untied}
+
+# Since the interrupts are not level triggered, we don't use scr_q register in rv_plic_gateway. So,
```

Comment:
This statement doesn't make sense. Interrupts _are_ level-triggered for all sources in our use of rv_plic. Is there a stray "not" in there?

Reply:
Thanks for flagging this out. That was a mistake.